packages/contracts/src/framework/plugin/repo/PluginRepo.sol

```diff
@@ -129,6 +114,22 @@ contract PluginRepo is
         _grant(address(this), initialOwner, UPGRADE_REPO_PERMISSION_ID);
     }
 
+    /// @notice Initializes the pluginRepo after an upgrade from a previous protocol version.
+    /// @param _previousProtocolVersion The semantic protocol version number of the previous DAO implementation contract this upgrade is transitioning from.
+    /// @param _initData The initialization data to be passed to via `upgradeToAndCall` (see [ERC-1967](https://docs.openzeppelin.com/contracts/4.x/api/proxy#ERC1967Upgrade)).
+    /// @dev This function is a placeholder until we require reinitialization.
+    function initializeFrom(
+        uint8[3] calldata _previousProtocolVersion,
```

Comment:
Check if we can go without silencing the variable by deleting the variable name

Reply:
Yes, this works, but we want to have the var name there to also write proper NatSpec. Compilers are smart enough to see that this is not used and optimize it away.